### PR TITLE
Small edits

### PR DESCRIPTION
**Pull request by pollfly**

### CODE REVIEW

File: docs/guides/advanced/execute_remotely.md [hunk 1]
```diff
@@ -52,8 +52,7 @@ Logger.current_logger().report_scalar(
 )
 ```    
 
-These scalars can be visualized in plots, which appear in the ClearML web UI, in the experiment's
-page **>** **SCALARS**. 
+These scalars can be visualized in plots, which appear in the ClearML web UI, in the experiment's **SCALARS**.
```
Comment (ainoam) [inline docs/guides/advanced/execute_remotely.md:55]: @pollfly scalars **tab**?

File: docs/guides/frameworks/pytorch/tensorboard_toy_pytorch.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ associated with the `examples` project.
 
 ## Debug Samples
 
-The debug sample images appear according to metric, in the experiment page in the **ClearML web UI** under **DEBUG SAMPLES**.
+The debug sample images appear according to metric, in the experiment's **DEBUG SAMPLES**.
```
Comment (ainoam) [inline docs/guides/frameworks/pytorch/tensorboard_toy_pytorch.md:12]: @pollfly debug samples **tab**?

File: docs/guides/frameworks/tensorboardx/video_tensorboardx.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ the `examples` project.
 ## Debug Samples
 
 ClearML automatically captures the video data that is added to the `SummaryWriter` object, using the `add_video` method. 
-The video appears in the experiment page in the ClearML web UI under **DEBUG SAMPLES**.
+The video appears in the experiment's **DEBUG SAMPLES**.
```
Comment (ainoam) [inline docs/guides/frameworks/tensorboardx/video_tensorboardx.md:15]: @pollfly debug samples **tab**?

File: docs/guides/advanced/execute_remotely.md (hunk 1 above)
Comment (pollfly) [reply docs/guides/advanced/execute_remotely.md]: added "tab"s, also in new commit switched page >>> tab when describing experiment sections

File: docs/guides/frameworks/megengine/megengine_mnist.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ The example script does the following:
 ## Hyperparameters
 
 ClearML automatically logs command line options defined with `argparse`. They appear in the experiment's **CONFIGURATIONS** 
-page under **HYPER PARAMETERS** **>** **Args**.
+tab under **HYPER PARAMETERS** **>** **Args**.
```
Comment (ainoam) [inline docs/guides/frameworks/megengine/megengine_mnist.md:18]: @pollfly which tab? -> Configuration tab

Comment (pollfly) [reply docs/guides/frameworks/megengine/megengine_mnist.md:18]: changed every instance (except release notes)

**Pull request merged**